images/capi/ansible/roles/providers/tasks/vmware.yml

```diff
@@ -64,6 +64,21 @@
     state: absent
   when: ansible_os_family == "VMware Photon OS"
 
+- name: Make sure all cloud init services are enabled and running
+  service:
+    name: "{{ item }}"
+    state: started
+    enabled: yes
+  with_items:
+    - cloud-final
+    - cloud-config
+    - cloud-init
+    - cloud-init-local
+  when: ansible_os_family == "RedHat"
```

Comment:
I'm wondering a few things here...

it seems to me that the goal here is "make sure all the cloud-init services are enabled". We found on RHEL, that after install some weren't enabled by default. But I'm wondering if the solution can't apply more globally. Would it be reasonable to put this logic in `roles/providers/tasks/main.yml` and make sure this works for all distros?

Do all distros have all four services? Or is the naming/partitioning of these services OS/distro specific?

And, like the other commit, I think we could probably pull this into it's own PR against the the non-fork of this repo. There's nothing RHEL-specific in this -- this particular code would run on CentOS as well, and i'm wondering if we can run it against all.

This would also fit better with our newly discussed goal of being more explicit about making sure installed services are enabled and running rather than relying on packaging default behaviors.